apps/root/src/pages/earn/portfolio/frame/index.tsx

```diff
@@ -7,16 +7,46 @@ import { EARN_PORTFOLIO } from '@constants/routes';
 import NetWorth from '@common/components/net-worth';
 import { WalletOptionValues, ALL_WALLETS } from '@common/components/wallet-selector';
 import EarnPortfolioFinancialData from '../components/financial-data';
+import StrategiesTable from '@pages/earn/components/strategies-table';
+import { StrategiesTableVariants } from '@state/strategies-filters/reducer';
+import useEarnPositions from '@hooks/earn/useEarnPositions';
+import { portfolioColumnConfigs } from '@pages/earn/components/strategies-table/components/columns';
+import { EarnPosition, StrategyId } from 'common-types';
+import useFilteredStrategies from '@hooks/earn/useFilteredStrategies';
+
+const variant = StrategiesTableVariants.USER_STRATEGIES;
 
 const EarnPortfolioFrame = () => {
   const [selectedWalletOption, setSelectedWalletOption] = React.useState<WalletOptionValues>(ALL_WALLETS);
+  const { userStrategies, hasFetchedUserStrategies } = useEarnPositions();
 
   const dispatch = useAppDispatch();
 
   React.useEffect(() => {
     dispatch(changeRoute(EARN_PORTFOLIO.key));
   }, []);
 
+  const groupedPositionsByStrategy = React.useMemo(() => {
```

Comment:
I'm kinda on the fence on this, since it's going to re-render EVERYTHING when any strategy changes. We can have a wrapper component for the StrategiesTable that just uses the hook internally so only that component re-renders